doc/cmd/preprocrepos.md

```diff
@@ -0,0 +1,26 @@
+# Preprocrepos command
+
+This command allows you to preprocess your data before passing it to any command you need.
+
+## Preprocrepos pipeline:
```

Comment:
This section needs command line examples for each point you mention below

Reply:
They will be in a separate file. 
I will add a links here